app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
It's what we do today, and I'm OK with it until we figure out how to improve nightly documentation (like copying it earlier).

I'd prefer to link to non-existent 1.9 documentation than try to fix links to 1.9 when the version it uses in the URLs is still 1.8...